app/assets/javascripts/components/story/StorySelect.js

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
 
 const renderOptions = (options, blank) => ([
-  <option value='' key={options.length}> {blank} </option>,
-  options.map((estimatePoints, i) =>
-    <option value={estimatePoints} key={i}>
-      {estimatePoints}
-    </option>
+  blank ? <option value='' key={options.length}> { blank } </option> : null,
```

Comment:
You probably can use just `blank && <option value='' key={options.length}> { blank } </option>` here.